File: app/src/main/java/com/example/nasa/data/api/NasaApi.java
package com.example.nasa.data.api;

import com.example.nasa.data.model.APODJson;

import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface NasaApi {

    @GET("planetary/apod")
    Single<APODJson> getAstronomyPicture(@Query("date") String date);
}

File: data/src/main/java/com/example/data/api/NasaApi.kt
package com.example.data.api

import com.example.data.model.APODJson
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface NasaApi {
    @GET("planetary/apod")
    fun getAstronomyPicture(@Query("date") date: String): Single<APODJson>
}